app/payment-link-v2/amount/amount.njk

```diff
@@ -0,0 +1,52 @@
+{% extends "../../views/layout.njk" %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/inset-text/macro.njk" import govukInsetText %}
+{% from "govuk/components/input/macro.njk" import govukInput %}
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+
+{% block pageTitle %}
+  {{ __p('paymentLinksV2.amount.enterAmountToPay') }} - {{ serviceName }}
```

Comment:
Probably makes sense to change {{ serviceName }} to be the payment link name in line with replacing that in the service banner.

Reply:
Done - thanks